paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -0,0 +1,99 @@
+package com.stripe.android.paymentsheet
+
+import android.app.Activity
+import androidx.activity.result.ActivityResultCaller
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import com.stripe.android.core.strings.ResolvableString
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.paymentsheet.model.PaymentSelection
+import kotlinx.coroutines.flow.StateFlow
+
+/**
+ * This interface handles the process of confirming a [StripeIntent]. This interface can only handle confirming one
+ * intent at a time.
+ */
+internal interface IntentConfirmationHandler {
```

Comment:
Do we need this interface? I think we can just use the implementation directly.

Reply:
I was thinking we can intent confirmation testing to the implementation while building a test handler for `PaymentSheetViewModel`. What do you think?